docs/inputs/conditional-input.md

```diff
@@ -8,91 +8,137 @@ sidebar_position: 5
-   `Checkbox == True` is not satisfied.
-   First select the **Checkbox Input**
+   <div style={{textAlign: 'center'}}>
+
+![img alt](/docs/inputs/conditional_input_hidden.png)
+
+   </div>
+
+Since we want to hide the **Slider** when the **Checkbox** is not checked, we select _Hidden_.
+
+**Step 3.** To create a **_condition_** you must define the 3 sub-components.  
+a. First select the **Checkbox** as the **Input** to be evaluated.
 
    <div style={{textAlign: 'center'}}>
 
-   ![img alt](/docs/inputs/conditional_input_selection.png)
+![img alt](/docs/inputs/conditional_input_selection.png)
 
    </div>
 
-   We want to check whether or not the **Checkbox** is TRUE, therefore select the '==' operator
+b. We want to evaluate whether or not the **Checkbox** is TRUE, therefore select the `==` operator
 
    <div style={{textAlign: 'center'}}>
 
-   ![img alt](/docs/inputs/conditional_operator_selection.png)
+![img alt](/docs/inputs/conditional_operator_selection.png)
 
    </div>
 
-   Finally, the value TRUE is selected
+c. Finally, the value TRUE is selected
 
    <div style={{textAlign: 'center'}}>
 
-   ![img alt](/docs/inputs/conditional_value_selection.png)
+![img alt](/docs/inputs/conditional_value_selection.png)
 
    </div>
 
 :::caution
 When editing an input, don't forget to click on **APPLY CHANGES** at the bottom of the screen to save your work
 :::
 
-## Test it out for youself
+## Test it out for yourself
 
 Test out the _**condition**_ we created above!  
 :::tip
-If the Checkbox Input is TRUE (checked) the Slider will be visible.
+If the **Checkbox** is TRUE (checked) the **Slider** will be visible.
 :::
 
 <div style={{width: "100%", height: "600px", overflow: "hidden"}}><iframe src="https://mecsimcalc.com/app/8755703/premium_conditional_input" width="100%" height="100%" title="MecSimCalc" style={{position:"relative", left:"-45px", top:"-48px"}} frameBorder="0"></iframe></div>
 
 ## Additional
 
-:information_source: **Note:** We currently do not offer creating conditions based on the following input types: date, file, rich text.
+:information_source: **Note:** We currently do not offer creating **_conditions_** based on the following **Input Types**: _Date Time_, _File_, _Rich Text_.
 
 ### Adding a condition
 
-To add a **_condition_**, simply click the plus button. There can be as many **_conditions_** as you want.
+To add a **_condition_**, click the ![img alt](/docs/inputs/add_condition.png) button. There can be as many **_conditions_** as you want!
 
    <div style={{textAlign: 'center'}}>
 
 ![img alt](/docs/inputs/conditional_add_rule.png)
 
    </div>
+
+### Combining Conditions
+
+You can combine **_conditions_** using `AND` or `OR`.
+
+#### Order of operations
+
+`AND` takes precedence over `OR`.  
+E.g. `A OR B AND C` is evaluated as `A OR (B AND C)`.
```

Comment:
Would something like this also help:

A OR B AND C is evaluated as follows:

1. B AND C is computed, resulting in the binary result D.
2. A OR D is computed, resulting in the binary result E.